test/e2e/monitoring_stack_controller_test.go

```diff
@@ -45,62 +45,63 @@ func assertCRDExists(t *testing.T, crds ...string) {
 func TestMonitoringStackController(t *testing.T) {
 	assertCRDExists(t,
 		"prometheuses.monitoring.rhobs",
+		"alertmanagers.monitoring.rhobs",
+		"podmonitors.monitoring.rhobs",
```

Comment:
any reason we didn't add `servicemonitors`?

Reply:
Good question :) .. The initial implementation was to create PodMonitor and to only deploy prometheus pod but I later change it to `servicemonitor` so, yes, we should assert that servicemonitor is present as well.